packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -201,82 +269,41 @@ export default class LocusInfo extends ChildEmitter {
-      console.log('Delta participants came through');
-    }
-    else {
-      console.log('Full locus participants came through');
-    }
-    this.emitChanges(EVENTS.LOCUS_INFO_UPDATE_PARTICIPANTS, {
+  updateParticipants(participants) {
+    this.emitGlobal(EVENTS.LOCUS_INFO_UPDATE_PARTICIPANTS, {
       participants,
       selfIdentity: this.parsedLocus.self.selfIdentity,
       selfId: this.parsedLocus.self.selfId,
       hostId: this.parsedLocus.host.hostId
     });
   }
 
-  // // TODO: check if this causes any perfomance issue
-  // findChangedParticipants(newParticipants) {
-  //   // TODO: Handle case where if a user got added and removed then find the one added or removed
-  //   if (newParticipants.length < 25) {
-  //     const changedParticipants = [];
-  //     this.participants.forEach((participant, index) => {
-  //       // find if participant is in the array
-  //       // find if any of the participant  got missed
-  //       if (participant && !isEqual(participant, newParticipants[participant.id])) {
-  //         this.participants[index] = newParticipants[participant.id];
-  //         changedParticipants[participant.id] = participant;
-  //       }
-  //     });
-  //     return changedParticipants;
-  //   }
-  //   // If more then 25 and we got fullLocus send Everything
-  //   return this.participants;
-  //   // TODO: find an alternate way or just replace the whole object
-  //   // Tell customers that more then 25 we will just send an update
-  // }
-
   updateControls(controls) {
     if (controls && !isEqual(this.controls, controls)) {
       this.controls = controls;
-      console.info('LocusInfo:controls got updated');
     }
   }
 
   updateConversationUrl(conversationUrl) {
     if (conversationUrl && !isEqual(this.conversationUrl, conversationUrl)) {
       this.conversationUrl = conversationUrl;
-      this.emitChanges(EVENTS.CONVERSATION_URL_UPDATE, conversationUrl);
-      console.info('LocusInfo:conversationUrl got updated');
+      this.emitGlobal(EVENTS.CONVERSATION_URL_UPDATE, conversationUrl);
     }
   }
 
   updateCreated(created) {
     if (created && !isEqual(this.created, created)) {
       this.created = created;
-      console.info('LocusInfo:created got updated');
     }
   }
 
   updateFullState(fullState) {
     if (fullState && !isEqual(this.fullState, fullState)) {
-      // TODO: check what exactly is needed
-      // active: true
-      // count: 1
-      // lastActive: "2019-02-01T16:52:26.764Z"
-      // locked: false
-      // sessionIds: []
-      // state: "ACTIVE"
-      // type: "CALL"
-
-      // {
-      //   'CALL'; // For 1:1 calls the type is CALL
-      //   'MEETING'; // When Guest added the type is meeting
-      //   'CALL'; // When guest user leaves
-      // }
+      const result = FullState.getFullState(this.fullState, fullState);
+
+      this.updateMeeting(result.current);
+      this.parsedLocus.fullState = result.current;
       this.fullState = fullState;
```

Comment:
is result.current different than full state? I basically have no idea what is going on in this method. some code comments at the top couldn't hurt